vulture/core.py

```diff
@@ -176,8 +181,9 @@ def scavenge(self, paths):
 
         for module in included_modules:
             self.log('Scanning:', module)
+            module_string = _get_stdlib_whitelist()
             try:
-                module_string = read_file(module)
+                module_string += read_file(module)
```

Comment:
This seems to be highly inefficient!
Can we use `pkg_resources`

We can then change `_get_stdlib_whitelist` to:
```python
def _get_stdlib_whitelist():
    """
    Returns absolute path of the `stdlib` whitelist.
    """
    return pkg_resources.resource_filename('vulture', 'whitelists/stdlib.py')
```